connection/connection_params.ts

```diff
@@ -47,19 +47,26 @@ export interface TLSOptions {
   /**
    * If TLS support is enabled or not. If the server requires TLS,
    * the connection will fail.
+   *
+   * Default: `true`
    */
   enabled: boolean;
   /**
    * This will force the connection to run over TLS
    * If the server doesn't support TLS, the connection will fail
    *
-   * default: `false`
+   * Default: `false`
    */
   enforce: boolean;
   /**
-   * A custom CA file to use for the TLS connection to the server.
+   * A list of root certificates that will be used in addition to the default
+   * root certificates to verify the server's certificate.
+   *
+   * Must be in PEM format.
+   *
+   * Default: `[]`
    */
-  caFile?: string;
+  caCerts?: string[];
```

Comment:
Rename to `ca_certificates`

Reply:
This code base already uses snake case broadly, and isn't camelcase being removed as a standard on deno lint anyway?

Reply:
It might use snake_case, but in the public API it uses it inconsistently with camelCase: https://doc.deno.land/https/deno.land/x/postgres@v0.13.0/mod.ts. For example `ClientOptions.applicationName` is camelCase. Also **all** APIs in the JS standard library (for complex examples, see `Intl`) are camelCase, and all Web APIs use camelCase too. camelCase is the defacto standard for JS APIs.

I don't particularly care what casing modules use internally, but I think providing a **public** API that looks approachable for users of the Web platform (which are familiar with camelCase only) is super important.

Reply:
I'll think about it. How about `caCertificates` in the meanwhile? What I really really don't want in the public API is unnecessary abbreviations

Reply:
Sounds good to me!